### DISCUSSION TITLE
Gravity flak shell's gravity pull is affected by Europa's gravity

### DISCUSSION THREAD

**Discussion by Xeavas** [FakeFishGames/Barotrauma#14974] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The gravity field is affected by gravity

### Reproduction steps

1. Test mode R-29 in submarine editor
2. Spawn gravity flak shell box
4. lock y (optional) & disableenemyai
3. Spawn Bonethreshers near flak and shoot them with gravity shells

https://github.com/user-attachments/assets/7123885e-de9c-4ee4-a845-628c6834deaa

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly, fix coming in the next hotfix update. Closing discussion.
